packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/in_app_purchase_storekit.h

```diff
@@ -0,0 +1,15 @@
+//
+//  in_app_purchase_storekit.h
+//  Pods
+//
+//  Created by Louise Hsu on 4/11/24.
+//
+
+#ifndef in_app_purchase_storekit_h
+#define in_app_purchase_storekit_h
+
+#if __has_include("in_app_purchase_storekit-umbrella.h")
```

Comment:
This has to be here b/c of this cocoapods issue -
https://github.com/CocoaPods/CocoaPods/issues/3767

Without this file, the generated "in_app_purchase_storekit-Swift.h" will keep trying to import an "in_app_purchase_storekit.h" which doesn't exist.

Reply:
can you add a comment there?